.github/workflows/publish-providers.yml

```diff
@@ -1,6 +1,14 @@
 name: publish-providers
 
 on:
+  push:
+    branches:
+      - main
+      - cd/fix-publish-workflows
+    paths:
+      - 'packages/mcp-provider-api/**'
+      - 'packages/mcp-provider-dx-core/**'  
+      # - 'packages/mcp-provider-code-analyzer/**'
```

Comment:
Could we change this to `'packages/mcp-provider-*/**`? That way new teams do not need to add new paths here

Reply:
> So instead we need to uncomment - 'packages/mcp-provider-code-analyzer/** here and then add some documentation on adding a new provider

it's commented out on purpose, the automated release is only enabled for the provider-api and dx-core modules because we own them but code-analyzer might want to have their own release cadence (manual release).

For instance, CA folks might be working on features for 2 weeks and merging stuff into `main` but without releasing that package, so if we need to trigger a server release during that time they are safe because `@salesforce/mcp` will be referring to their last published npm pkg.